### ISSUE TITLE
aws_redshift_security_group forces new resource with an update to the SG

### ISSUE THREAD

**Issue by pmvilaca** [hashicorp/terraform#5707]
If we add or remove an ingress rule to a aws_redshift_security_group resource, it forces a new resource. Apparently, there is no need for that because we can add/remove rules to a security group through the console.

```
-/+ aws_redshift_security_group.default
    description:                                "SG that contains the default rules for redshift clusters"
    ingress.#:                                  "2" => "1" (forces new resource)
    ingress.1542823501.cidr:                    "192.168.0.0/16" => "192.168.0.0/16"
    ingress.1542823501.security_group_name:     "" => "<computed>"
    ingress.1542823501.security_group_owner_id: "" => "<computed>"
    ingress.2720885756.cidr:                    "" => ""
    name:                                       "redshift-default-sg" => "redshift-default-sg"
```

**Comment by stack72** [comment]
Hi @pmvilaca,

thanks for reporting this issue. FWIW, the redshift work was started before the AWS SDK added the update work. I just had to open another PR that added the ability to modify a cluster for a specific parameter

I will work on this and add it

Paul

**Comment by joshuaspence** [comment]
I think that the same issue occurs with RDS security group rules too.

**Comment by stack72** [comment]
Fixed via #5939
